test/extensions/filters/http/grpc_http1_reverse_bridge/reverse_bridge_integration_test.cc

```diff
@@ -173,50 +173,5 @@ TEST_P(ReverseBridgeIntegrationTest, EnabledRoute) {
   ASSERT_TRUE(fake_upstream_connection_->waitForDisconnect());
 }
 
-TEST_P(ReverseBridgeIntegrationTest, EnabledRouteBadContentType) {
```

Comment:
FWIW I don't see the harm of leaving this test.

Reply:
I was thinking that it wouldn't pass without the other PR but I'm not sure if that's true, I'll try adding it back

Reply:
I was able to get it working, seems like it just tested an existing feature so it passes now and should nicely serve as a regression test to make sure that we're not breaking this feature with the change to use sendLocalReply in the future